### ISSUE TITLE
ReScience (R)evolution

### ISSUE THREAD

**Issue by rougier** [ReScience/ReScience#79]
Dear @ReScience/associate-editors, @ReScience/editors, 

We (@khinsen and me) have submitted an editorial at [this address](https://github.com/rescience-c/submissions/issues/1) that propose some changes for the journal in order to make the life of everybody easier. You can read the paper and wait for the review to start to comment on the proposed ideas. The most noticeable change is a slight name modification (ReScience C instead of ReScience) in order to prepare the  future ReScience X.

Nicolas

**Comment by otizonaizit** [comment]
Hi @ReScience/reviewers @ReScience/associate-editors @ReScience/editors !

You are all welcome to participate in reviewing the newly submitted editorial: https://github.com/rescience-c/reviews/issues/1
The editorial proposes some important changes to the submission/review/publication to the ReScience journal family and also a new layout.

I am looking forward to a fruitful discussion!
Tiziano

**Comment by labarba** [comment]
If ReScience is interested in using the JOSS publication workflow, I can heartily recommend it—out bot `whedon` is super-powered. Check out this one example: I published this paper today, after all review/submission/archive deposit steps were done, in _just minutes_, doing:

`@whedon accept`

... then check that everything is correct, sanity check on the Crossref deposit and PDF. Then,

`@whedon accept deposit=true`

... and the paper is published!! Close the issue, and done. Celebrate. It's awesome.

**Comment by rougier** [comment]
That's the idea. We're actually working with @trallard (Tania Allard) to see how much of `@whedon` we can use. It is not clear yet but we'll have a better idea of the workflow after this submission has been reviewed. I will then specify  what is actually needed from the bot.

**Comment by trallard** [comment]
Yes, we have had some chats around this and it would be really exciting to get `whedon` working for Rescience

**Comment by tpoisot** [comment]
Hi all

We're about to start a MS for ReScience as part of a graduate class, should we still use the former submission process or should we volunteer to test drive this one?

**Comment by rougier** [comment]
That's great ! The new process is supposed to be more simple but a real test would help to decide if it is the case. What is not yet clear to me is whether I just transform the old website or if we migrate everything to this new website (not sure about how to migrate editor/reviewers).

If you intend to have many submission, we could also test the "special issue" stuff.
In the meantime, we also need to decide if the proposed changes are approved / not approved by people.

@otizonaizit  Can you setup a simple 👍 / 👎 poll to move things forward ?

**Comment by tpoisot** [comment]
@rougier we'll have two submissions, so not quite a special issue -- for now, we'll just setup projects as we normally would, and we'll reformat the markdown files if needed in the correct format.
